extensions/librdkafka/tests/ConsumeKafkaTests.cpp

```diff
@@ -586,4 +598,19 @@ TEST_CASE_METHOD(ConsumeKafkaContinuousPublishingTest, "ConsumeKafka can spend n
   //  I tried adding a wait time for more than "session.timeout.ms" inbetween tests, but it was not sufficient
 }
 
+TEST_CASE_METHOD(ConsumeKafkaPropertiesTest, "ConsumeKafka can communicate with the broker via SSL.", "[ConsumeKafka][Kafka][SSL]") {
+  auto run_tests = [&] (const std::vector<std::string>& messages_on_topic) {
+    single_consumer_with_plain_text_test(true, {}, messages_on_topic, NON_TRANSACTIONAL_MESSAGES, {}, "localhost:9093", ConsumeKafka::SECURITY_PROTOCOL_SSL, "ConsumeKafkaTest", {}, {}, "test_group_id", {}, {}, {}, {}, {}, {}, "1", "2 sec", "60 sec"); // NOLINT
+  };
+  const auto get_current_timestamp = [] {
+    const std::time_t result = std::time(nullptr);
+    std::stringstream time_stream;
+    time_stream << std::asctime(std::localtime(&result));
+    return time_stream.str();
+  };
```

Comment:
I'd prefer a C++'y way of doing this, using the chrono, date and date-tz libraries. In addition to the below suggestion, you need to link the date and date-tz targets to the test and include "date/date.h" and "date/tz.h".
```suggestion
  const auto get_current_timestamp = [] {
    return date::format("%a %b %e %H:%M:%S %Y", date::make_zoned(date::current_zone(), std::chrono::floor<std::chrono::seconds>(std::chrono::system_clock::now())));
  };
```

The C++20 version would look like this, but currently only MSVC implements the timezone and format support required:
```
  const auto get_current_timestamp = [] {
    return std::format("{:%a %b %e %H:%M:%S %Y}", std::chrono::zoned_time{std::chrono::current_zone(), std::chrono::floor<std::chrono::seconds>(std::chrono::system_clock::now())});
  };
```

If you don't want to introduce the dependencies just to save a few lines, that's also fine for me.

Reply:
I would prefer not to put too much additional effort into these tests, as there is already a [Jira ticket](https://issues.apache.org/jira/browse/MINIFICPP-1518) for removing them in the future, as it is only a manual test suite and the test cases are already implemented (or mostly implemented) in the integration test suite. After this PR is merged the parity between the manual and the automated test suites of the kafka extension should be evaluated and after the gaps are fixed, the manual suites can be removed.